me/index.php

```diff
@@ -1,24 +1,278 @@
 
 <?php
 require('../template/top.php');
-head('My Profile', true, true);
+head('Edit My Profile', true, true);
+
 ?>
 
+
+<head>
+    <style>
+        .profileName{
+            padding: 20px 20px 20px 20px;
+            text-align: center;
+            font-family: Ubuntu;
+            font-style: normal;
+            font-weight: bold;
+            font-size: 48px;
+            line-height: 55px;
+            color: #18AC5C;
+        }
+        .profileText{
+            text-align: center;
+            font-family: Ubuntu;
+            font-style: normal;
+            font-weight: normal;
+            font-size: 24px;
+            line-height: 28px;
+            color: #000000;
+        }
+        .center {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+            width: 50%;
+        }
+        .button{
+            display: inline-block;
+            background: #007A38;
+            border-radius: 6px;
+            margin-bottom: 25px;
+            padding: 2px 15px 2px 15px;
+            text-align: center;
+            font-family: Ubuntu;
+            font-style: normal;
+            font-weight: normal;
+            font-size: 22px;
+            color: #FFFFFF;
+            cursor: pointer;
+        }
+        .bottomButton{
+            position: relative;
+            width: 250px;
+            display: inline-block;
+            background: #007A38;
+            border-radius: 6px;
+            margin-bottom: 25px;
+            padding: 3px 15px 3px 15px;
+            text-align: center;
+            font-family: Ubuntu;
+            font-style: normal;
+            font-weight: normal;
+            font-size: 22px;
+            color: #FFFFFF;
+            cursor: pointer;
+        }
+        .orangeBox{
+            padding-top: 20px;
+            padding-bottom: 20px;
+            background: #EDEDED;
+            border-radius: 30px;
+            margin-bottom: 3%;
+        }
+        .picture{
+            height: 50px;
+            width: 50px;
+            cursor: pointer;
+        }
+    </style>
+    <script>
+        function addButton(){
+            document.getElementById("editButton").setAttribute("disabled", "true");
+
+            var buttonChange = document.createElement('button');

```

Comment:
Alex, the infinite button creation is probably because of this line. You're recreating the element every time `addButton()` is called. The button is never deleted, it's only disabled [here](https://github.com/untrobotics/website/blob/1b7944b09a4ad533bb3be67204e5081f8a8af1ee/me/index.php#L112).

You should probably have the "Save" button already in the HTML, or you should check that it exists before using `createElement()`